bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/SpectralDistribution.scala

```diff
@@ -0,0 +1,192 @@
+  case object PlanetaryNebula   extends LibraryNonStar("Planetary nebula (NGC7009: 100-1100nm)",  "Planetary-nebula")
+  case object PlanetaryNebula2  extends LibraryNonStar("Planetary nebula (IC5117: 480-2500nm)",   "Planetary-nebula2")
+  case object PlanetaryNebula3  extends LibraryNonStar("Planetary nebula (NGC7027)",              "Planetary-nebula-NGC7027")
+  case object StarburstGalaxy   extends LibraryNonStar("Starburst galaxy (M82)",                  "Starburst-galaxy")
+  case object PmsStar           extends LibraryNonStar("Pre-main sequence star (HD100546)",       "PMS-star")
+  case object GalacticCenter    extends LibraryNonStar("Galactic center",                         "Galactic-center")
+  case object Afgl230           extends LibraryNonStar("AFGL230 (M10II star, silicate absorp.)",  "afgl230")
+  case object Afgl3068          extends LibraryNonStar("AFGL3068 (Late N-type star)",             "afgl3068") // orig desc: "AFGL3068 (Late N-type star, optically thick carbon based dust)"
+  case object AlphaBoo          extends LibraryNonStar("Alpha Boo (K1.5III star)",                "alphaboo")
+  case object AlphaCar          extends LibraryNonStar("Alpha Car (F0II star)",                   "alphacar")
+  case object BetaAnd           extends LibraryNonStar("Beta And (M0IIIa star)",                  "betaand")
+  case object BetaGru           extends LibraryNonStar("Beta Gru (M5III star)",                   "betagru")
+  case object GammaCas          extends LibraryNonStar("Gamma Cas (B0IVe star)",                  "gammacas")
+  case object GammaDra          extends LibraryNonStar("Gamma Dra (K5III star)",                  "gammadra")
+  case object L1511Irs          extends LibraryNonStar("l1551irs (young stellar object)",         "l1551irs")
+  case object NGC1068           extends LibraryNonStar("NGC 1068 (Dusty active galaxy)",          "ngc1068")
+  case object NGC2023           extends LibraryNonStar("NGC2023 (Reflection Nebula)",             "ngc2023")
+  case object NGC2440           extends LibraryNonStar("NGC2440 (line dominated PN)",             "ngc2440")
+  case object OCet              extends LibraryNonStar("O Cet (M7IIIa Star, silicate emission)",  "ocet")
+  case object OrionBar          extends LibraryNonStar("Orion Bar (Dusty HII region)",            "orionbar")
+  case object Rscl              extends LibraryNonStar("rscl (N-type Dusty Carbon Star, SiC em.)","rscl")
+  case object Txpsc             extends LibraryNonStar("txpsc (N-type Visible Carbon Star)",      "txpsc")
+  case object Wr104             extends LibraryNonStar("WR 104 (Wolf-Rayet Star + dust)",         "wr104")
+  case object Wr34              extends LibraryNonStar("WR 34 (Wolf-Rayet Star)",                 "wr34")
+
+  /** All available non-star spectras. */
+  val values = List(
+    EllipticalGalaxy, SpiralGalaxy, QS0, QS02, OrionNebula, PlanetaryNebula, PlanetaryNebula2, PlanetaryNebula3,
```

Comment:
That's what I had to start with. I switched to ```case object```s because it makes it straight forward to reference a particular object for example in test fixtures. However, I have to admit, that's the only place (right now) where the ```case object```s are of use.